server/controllers/feeds/feeds.coffee

```diff
@@ -0,0 +1,27 @@
+CONFIG = require 'config'
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+root = CONFIG.fullPublicHost()
+items_ = __.require 'controllers', 'items/lib/items'
+error_ = __.require 'lib', 'error/error'
+RSS = require 'rss'
+limit = 100
+
+module.exports =
+  get: (req, res, next)->
+    feed = new RSS({"title":"New Public Items"})
+    { query } = req
+    items_.publicByDate limit
```

Comment:
yep, [publicListings](https://github.com/inventaire/inventaire/blob/6fa25ab1c80b3449edd856b7dbe8cd1ce3365dd4/server/controllers/items/lib/items.coffee#L41) seems the right function:
```coffee
{ user:userId } = query
# items_.publicListings expects an array of user ids
items_.publicListings [ userId ]
```